project/CelebornBuild.scala

```diff
@@ -1005,7 +1016,8 @@ object MRClientProjects {
           Dependencies.hadoopClientApi,
           Dependencies.hadoopClientRuntime,
           Dependencies.hadoopMapreduceClientApp
-        ) ++ commonUnitTestDependencies
+        ) ++ commonUnitTestDependencies,
+        dependencyOverrides += Dependencies.commonsCompress
```

Comment:
@cfmcgrady, @cxzl25, the `commons-compress` dependency only needs to override for `MRClientProjects`, instead of common setting. The `commons-compress` dependency is also used in `flink-1.19`.